packages/manager/modules/manager-pci-common/src/api/data/instance.ts

```diff
@@ -23,6 +23,9 @@ export type TInstance = {
   region: string;
   sshKeyId?: string;
   status: string;
+  statusGroup?: string;
+  regionName?: string;
+  flavorName?: string;
```

Comment:
if you check the api schema those properties does not exists :
https://eu.api.ovh.com/console/?section=%2Fcloud&branch=v1#get-/cloud/project/-serviceName-/instance

you have to add those props in the pci-workflow app using a new type but the types in pci-common should reflect the api schema